browser/brave_browser_main_extra_parts.cc

```diff
@@ -37,6 +39,15 @@ void RecordInitialP3AValues() {
                                             g_browser_process->local_state());
 }
 
+// Initializes callback for SystemRequestHandler
+void InitSystemRequestHandlerCallback() {
+  net::SystemRequestHandler::OnBeforeSystemRequestCallback
+      before_system_request_callback =
+      base::Bind(brave::OnBeforeSystemRequest);
```

Comment:
looks like a wrong spacing, `clang-format` could help

Reply:
Fixed. Installed `clang-format` for my text editor. Thanks for suggestion.